src/pages/installation.md

```diff
@@ -3,58 +3,67 @@ layout: ~/layouts/Main.astro
 title: Installation
 ---
 
-There are a few different ways to install
+There are a few ways to getting started with Astro. Outlined below are instructions on how to go about installing Astro either manually or by our Astro Installer.
 
-## Prerequisites
+## System Requirements
 
-- **Node.js** - `v12.20.0`, `v14.13.1`, `v16.0.0`, or higher.
-- **A text editor** - We recommend [VS Code](https://code.visualstudio.com/) with the [Astro extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode).
-- **A terminal** - Astro is mainly accessed by terminal command-line.
+- **Node.js** - [`v12.20.0`](https://nodejs.org/en/download/releases/), [`v14.13.1`](https://nodejs.org/en/download/),[ `v16.0.0`](https://nodejs.org/en/download/current/), or higher.
+- **A text editor** - We recommend [VS Code](https://code.visualstudio.com/) with our own [Astro extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) for the complete DX.
```

Comment:
Perhaps write out Developer Experience?

Reply:
tails it is, ill also get that amended bud